tests/features/event.feature

```diff
@@ -131,6 +131,9 @@ Feature: Event content creation
     And I fill in "Subject" with "EU financing"
     And I fill in "URL" with "http://ec.europa.eu"
     And I fill in "Link text" with "Website"
+    And I fill in "URL" with "http://example.com" in the "Social media links" region
```

Comment:
Can we do one single social media link thing here? For example URL for facebook, title for facebook and type for facebook?